sql/create_train_val.go

```diff
@@ -174,7 +168,7 @@ func namingTrainAndValDataset(origTable string) *trainAndValDataset {
 
 func releaseTrainAndValDataset(db *DB, ds *trainAndValDataset) error {
```

Comment:
Note that we should not drop the validation table after training because we may still need to use that table to run a full evaluation. Can run this function before the training start yet.

Reply:
I see that only hive will do the `DROP TABLE`, this does not affect current running cases.